route/note-route.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Note = require('../model/note');
+const response = require('../lib/response');
+
+var notePool = {};
+
+module.exports = function(router){
+  router
+  //Create New
+  .post('/api/note', function(req, res){
+    if(req.body){
+      const note = new Note(req.body.content);
+      notePool[note.id] = note;
+      return response(200, note)(res);
+    }
+    response(400, 'bad request')(res);
+  })
+
+  .get('/api/note', function(req, res){
+    const note = notePool[req.url.query.id];
+    if(note){
+      return response(200, note)(res);
+    }
+    response(404, 'not found')(res);
+  })
+  //Update Existing
+  .put('/api/note', function(req,res){
+    const note = notePool[req.body.id];
+    if(note){
+      notePool[req.body.id].content = req.body.content;
```

Comment:
This is great for lab 07, for lab 08 you'll want to implement an update function in your storage module's and use it here to get the new content into the appropriate note.  Please note, PUT functionality will not be necessary for this assignment.  That being said it is a good thing to understand how to do.